tests/runtime/strings/wasm_utf16.c

```diff
@@ -0,0 +1,40 @@
+#include <assert.h>
+#include <imports.h>
+#include <exports.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+char16_t BASIC_STRING[] = u"latin utf16";
+// 🚀 = 0xD83D 0xDE80
+// 𠈄 = 0xD840 0xDE04
+// 𓀀 = 0xD80C 0xDC00
+char16_t UNICODE_STRING[] = { 0xD83D, 0xDE80, 0xD83D, 0xDE80, 0xD83D, 0xDE80, ' ', 0xD840, 0xDE04, 0xD80C, 0xDC00 };
```

Comment:
Could this be `u"🚀🚀🚀 𠈄𓀀"`?

Reply:
I did try that but it didn't seem to work for some reason.